main.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function(){
+
+$("a").click(function(event){
```

Comment:
Try to be consistent in usage of single and double quotes. Here you use double quotes but further down you use single quotes.